src/libAtomVM/opcodesswitch.h

```diff
@@ -2054,6 +2055,27 @@ static const char *const try_clause_atom = "\xA" "try_clause";
                 NEXT_INSTRUCTION(next_off);
                 break;
             }
+                
+            case OP_SET_TUPLE_ELEMENT: {
+                int next_off = 1;
+                term new_element;
+                DECODE_COMPACT_TERM(new_element, code, i, next_off, next_off);
+                term tuple;
+                DECODE_COMPACT_TERM(tuple, code, i, next_off, next_off);
+                int position;
+                DECODE_INTEGER(position, code, i, next_off, next_off);
+
+                USED_BY_TRACE(new_element);
+                USED_BY_TRACE(tuple);
+                USED_BY_TRACE(position);
+                TRACE("set_tuple_element/2, new_element=%lu tuple=%lu position=%i\n", new_element, tuple, position);
```

Comment:
we should avoid to TRACE values here: tuple and new_element will be uninitialized during code loading phase so junk will be printed, furthermore that would make valgrind complain.

Reply:
Good point.  Fixed!